crates/pipeline-manager/src/bin/pipeline-manager.rs

```diff
@@ -98,9 +99,31 @@ async fn main() -> anyhow::Result<()> {
             .await
             .expect("Local runner main failed");
     });
+
+    let health_check = Arc::new(RwLock::new(None));
```

Comment:
`common_config` can be passed to the `regular_health_check` function to form these URLs, as they are not used elsewhere. The main() does not need to contain it.